config/v1/techpreview.dns.testsuite.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "[Stable] DNS"
```

Comment:
Should this be `[TechPreview]`? (note: i'm not sure myself what this `Stable` refers to.